test/integration/proxy_proto_integration_test.cc

```diff
@@ -92,6 +92,32 @@ TEST_P(ProxyProtoIntegrationTest, RouterProxyUnknownLongRequestAndResponseWithBo
   testRouterRequestAndResponseWithBody(1024, 512, false, &creator);
 }
 
+// Test that %DOWNSTREAM_DIRECT_REMOTE_ADDRESS*% returns the direct address, and
```

Comment:
nit: extra asterisk here

Reply:
I added that to note that it also applies to DOWNSTREAM_DIRECT_REMOTE_ADDRESS_WITHOUT_PORT, but maybe I should just enumerate the two.